src/base/bittorrent/torrenthandle.cpp

```diff
@@ -816,7 +822,10 @@ void TorrentHandle::updateState()
             m_state = isSeed() ? TorrentState::PausedUploading : TorrentState::PausedDownloading;
     }
     else {
-        if (m_session->isQueueingSystemEnabled() && isQueued() && !isChecking()) {
+        if (isMoveInProgress()) {
+            m_state = TorrentState::Moving;
```

Comment:
I don't understand why paused torrent can't be in moving state... Can't we move it? I don't see such restrictions in qBittorrent code.